src/db/migrations/20190728221104-create-upVote.js

```diff
@@ -0,0 +1,49 @@
+import Debug from 'debug';
+
+const debug = Debug('dev');
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    try {
+      await queryInterface.createTable('UpVote', {
```

Comment:
Rename the file to `...-create-upvote` and rename the table to `Upvotes`

Reply:
done